File: codes/Haskell/p053.hs
{- 
 - Solution to Project Euler problem 53
 - Copyright (c) Sankha Saha. All rights reserved.
 - 
 -  
 -   
 -}

import qualified EulerLib


main = putStrLn (show ans)
ans = sum [1 | n <- [1..100], r <- [0..n], EulerLib.binomial n r > 10^6]

File: codes/python/p053.py
# 
# Solution to Project Euler problem 53
# Copyright (c) Sankha Saha. All rights reserved.
# 
#  
#   
# 

import math


def compute():
	ans = sum(1
		for n in range(1, 101)
		for k in range(0, n + 1)
		if math.comb(n, k) > 1000000)
	return str(ans)


if __name__ == "__main__":
	print(compute())
